src/Nix/Expr/Types.hs

```diff
@@ -201,7 +201,17 @@ instance IsString (Params r) where
 
 -- | 'Antiquoted' represents an expression that is either
 -- antiquoted (surrounded by ${...}) or plain (not antiquoted).
-data Antiquoted (v :: *) (r :: *) = Plain !v | EscapedNewline | Antiquoted !r
+data Antiquoted (v :: *) (r :: *)
+  = Plain !v
+  | EscapedNewline
+  -- ^ 'EscapedNewline' corresponds to the special newline form
+  --
+  -- > ''\n
+  --
+  -- in an indented string. It is equivalent to a single newline character:
+  --
+  -- > ''''\n'' === "\n"
```

Comment:
Is there a better way to express the equivalence of two Nix expressions?

For I now I just picked `===` because I'm familiar with that from Dhall. `≡` would look even nicer.